### PR TITLE
Get rid of the explicit JTH dependency so that the tests can run more reliably in PCT

### PR DESCRIPTION
**Pull request by oleg-nenashev**
@reviewbybees @jglick

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,15 +44,14 @@
 
     <issueManagement>
         <system>GitHub</system>
-        <url>https://github.com/janinko/ghprb/issues</url>
+        <url>https://github.com/jenkinsci/ghprb/issues</url>
```
Comment (jglick) [inline pom.xml]: no

Comment (jglick) [reply pom.xml]: There are 168 open in the first, 92 in your replacement. Really JIRA should be used and both closed, but this is an issue for the plugin maintainer to sort out.

Comment (oleg-nenashev) [reply pom.xml]: yes, "-plugin" is missing

**Comment by samrocketman** [comment]
Superseded by #615.

**Pull request closed without merging**